src/icmp/IcmpPinger.h

```diff
@@ -56,4 +56,3 @@ extern IcmpPinger control;
 #endif
 
 #endif
-
```

Comment:
Please format the changed sources if you can.

Reply:
done